sops/__init__.py

```diff
@@ -820,6 +840,10 @@ def encrypt(value, key, aad=b'', stash=None, digest=None):
     if digest:
         digest.update(value)
 
+    # if we don't want to encrypt, then return the value
+    if unencrypted:
+        return value
+
```

Comment:
I think a cleaner approach would be to do a first pass on the tree to calculate the digest, then do a second pass to encrypt it. It's beyond the scope of this PR though, so leave it as it is.